### PR TITLE
Extend resource pool configuration with resource prefix option

### PR DESCRIPTION
**Pull request by almaslennikov**
Resource configuration now might optionally
contain a resource prefix.
The previous behavior was to always add "rdma"
prefix to all resources. Now it's a default
option and might be changed if "resourcePrefix"
is provided in the config

Signed-off-by: amaslennikov <amaslennikov@nvidia.com>

### CODE REVIEW

**Comment by adrianchiris** [comment]
>Resource configuration now might optionally
contain a resource prefix.

Extend resource pool configuration with resource prefix option

File: README.md [hunk 1]
```diff
@@ -106,12 +107,13 @@ Notes:
...
+| Field            | Required | Description                                                                                                                       |     Type         | Default value | Example                                                 |
+|------------------|----------|-----------------------------------------------------------------------------------------------------------------------------------|------------------|---------------|---------------------------------------------------------|
+| "resourceName"   | Y        | Endpoint resource name. Should not contain special characters, must be unique in the scope of the resource prefix                 | string           | -             | "hca_shared_devices_a"                                  |
+| "resourcePrefix" | N        | Endpoint resource prefix. Should not contain special characters, must be unique in the scope of the resource prefix               | string           | "rdma"        | "example_prefix"                                        |
```
Comment (adrianchiris) [inline README.md:113]: > must be unique in the scope of the resource prefix

not needed / not applicable

Comment (almaslennikov) [reply README.md]: updated

File: README.md [hunk 2]
```diff
@@ -75,49 +76,63 @@ kubectl create -f <rdma-app.yaml>
...
-    "resourceName": "hca_shared_devices_a",
-    "rdmaHcaMax": 1000,
-    "devices": ["ib0", "ib1"]
-  },
+  "configList": [
+    {
+      "resourceName": "hca_shared_devices_a",
+      "resourcePrefix": "example_prefix",
+      "rdmaHcaMax": 1000,
+      "devices": [
```
Comment (adrianchiris) [inline README.md:90]: any chance to keep the old (compact) format here and below  :)

**Review by adrianchiris — APPROVED** [review]

**Pull request merged**